pkg/ddl/foreign_key.go

```diff
@@ -161,8 +161,9 @@ func checkTableForeignKeysValid(sctx sessionctx.Context, is infoschema.InfoSchem
 	}
 
 	referredFKInfos := is.GetTableReferredForeignKeys(schema, tbInfo.Name.L)
+	ctx := infoschema.WithRefillOption(context.Background(), false)
```

Comment:
seems it's always false right now

Reply:
Yes, we do not fill cache for TableByName in some place like in a loop that may visit a lot tables.